pkg/controller/statusmanager/status_manager.go

```diff
@@ -249,13 +441,13 @@ func (status *StatusManager) set(reachedAvailableLevel bool, conditions ...operv
 		}
 
 		if isNotFound {
-			if err := status.client.Create(context.TODO(), co); err != nil {
+			if err := status.client.ClientFor(cnoclient.DefaultClusterName).CRClient().Create(context.TODO(), co); err != nil {
```

Comment:
You can just write `status.client.Default()` in this case.

Reply:
will update to use `""`

Reply:
Updated